packages/Chem/src/rdkit-service/rdkit-service-worker-similarity.ts

```diff
@@ -1,6 +1,12 @@
-import {RdKitServiceWorkerBase} from './rdkit-service-worker-base';
-import {defaultMorganFpLength, defaultMorganFpRadius, Fingerprint} from '../utils/chem-common';
-import {RDModule} from '@datagrok-libraries/chem-meta/src/rdkit-api';
+import { RdKitServiceWorkerBase } from './rdkit-service-worker-base';
+import { defaultMorganFpLength, defaultMorganFpRadius, Fingerprint } from '../utils/chem-common';
+import { RDModule } from '@datagrok-libraries/chem-meta/src/rdkit-api';
+import { getMolSafe, IMolContext } from '../utils/mol-creation_rdkit';
+
+export interface IFingerprint {
```

Comment:
Can't we infer length from the array?

Reply:
Yes, we definitely can. I removed length both with IFingerprint interface. We do not need it anymore.